isis/tests/FunctionalTestsCubeatt.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+
+#include "Cube.h"
+#include "Fixtures.h"
+
+#include "cubeatt.h"
+#include "gmock/gmock.h"
+
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cubeatt.xml").expanded();
+
+// Tests setting output attributes: bit type and range
+TEST_F(SmallCube, FunctionalTestCubeattBitttypeAndRange) {
+  QString cubePath = tempDir.path() + "/bitTypeCubeatt.cub+8bit+0.0:1.0";
+
+  QVector<QString> args = {"from=" + testCube->fileName(), "to=" + cubePath};
+  UserInterface options(APP_XML, args);
+  cubeatt(options);
+
+  Cube outputCube(cubePath);
+
+  // Check attributes: pixel type, storage format, label format, storage order, pixel range, bands
+  EXPECT_EQ(outputCube.pixelType(), PixelType::UnsignedByte);
+  EXPECT_EQ(outputCube.format(), Cube::Format::Tile);
+  EXPECT_TRUE(outputCube.labelsAttached());
+  EXPECT_EQ(outputCube.byteOrder(), ByteOrder::Lsb);
```

Comment:
You shouldn't check every attribute in every test, you should only check the attributes that this test is working with. So, here I would not check format, attached label, or byte order.

Reply:
Sure, I can do that. My logic was that it was useful to test both the attributes that should change, and also the attributes that should remain the same.

Reply:
Oops, the functionality I just mentioned is definitely already being tested by the `noChange` test, nevermined.